korge/src/commonMain/kotlin/com/soywiz/korge/view/Sprite.kt

```diff
@@ -87,28 +87,56 @@ open class Sprite(
         }
     }
 
+    fun playAnimation(
+        spriteAnimation: SpriteAnimation? = currentAnimation,
+        spriteDisplayTime: TimeSpan = this.spriteDisplayTime,
+        startFrame : Int? = null,
+        reversed : Boolean = false) =
+        updateCurrentAnimation(
+            spriteAnimation = spriteAnimation,
+            spriteDisplayTime = spriteDisplayTime,
+            startFrame = startFrame ?: currentSpriteIndex,
```

Comment:
Try to avoid `Int? = null` here if possible. Change to:
* `: Int = -1`
* `startFrame ?: currentSpriteIndex` -> `if (startFrame >= 0) startFrame else currentSpriteIndex`

Rationale: `Int?` boxes integers adding overhead to the function and potentially adds pressure to the GC. Inside korlibs we are trying to reduce as much as possible all this overhead, specially in places that could potentially be called a lot of times (lots of sprites).

Reply:
can do that. Good to know